virtool/api/response.py

```diff
@@ -88,13 +88,20 @@ async def error_middleware(req: web.Request, handler: Callable):
     :param req: the incoming request
     :param handler: the next middleware to be executed
     """
+    logger = logging.getLogger()
     try:
+        logger.error("BEFORE HANDLE")
         return await handler(req)
 
     except web.HTTPException as exc:
+        logger.error("CAUGHT ERROR")
         data = {"id": "_".join(exc.reason.lower().split(" ")), "message": exc.text}
 
+        if exc.reason == "Not Found":
```

Comment:
Please add a comment to say why this is being done. I can see it being opaque in the future.